apps/frontend/src/libs/components/protected-route/protected-route.tsx

```diff
@@ -26,11 +37,22 @@ const ProtectedRoute = ({ children }: Properties): JSX.Element => {
 		);
 	}
 
-	return hasAuthenticatedUser ? (
-		<>{children}</>
-	) : (
-		<Navigate replace to={AppRoute.SIGN_IN} />
-	);
+	if (!hasAuthenticatedUser) {
+		return <Navigate replace to={AppRoute.SIGN_IN} />;
+	}
+
+	const hasRequiredPermission = authenticatedUser
+		? checkHasPermission(
```

Comment:
this check is no longer needed
```suggestion
	const hasRequiredPermission = checkHasPermission(
```

Reply:
I can't do that because I get a `Forbidden non-null assertion` error